.azure-pipelines/windows/basic-tests.yml

```diff
@@ -0,0 +1,21 @@
+#
+# Azure Pipelines configuration for building and testing react on Windows.
+#
+
+steps:
+  - script: yarn build
+    displayName: 'Build'
+
+  - script: set JEST_JUNIT_OUTPUT=.azure-pipelines/test-results/test-basic-windows.xml&&yarn test
```

Comment:
Think you can break this into two lines ... would be more clear what's happening